x-element.js

```diff
@@ -937,33 +941,33 @@ XElement.__propertyKeys = new Set(['type', 'attribute', 'input', 'compute', 'obs
 XElement.__serializableTypes = new Set([Boolean, String, Number]);
 XElement.__caseMap = new Map();
 
-const PROBLEMATIC_PROPERTY_NAMES = new Map();
-const PROBLEMATIC_ATTRIBUTES = new Map();
+const BAD_PROPERTY_NAMES = new Map();
```

Comment:
At a glance it's not clear what `bad` means. Is it synonymous with `RESERVED_PROPERTY_NAMES`?

Reply:
Just trying to cut PROBLEMATIC down. `INHERITED` was what I originally picked, but it's not quite right, the property `className` is inherited, but the property `class` is _problematic_ since it reflects to the _inherited_ attribute `class`.

Naming things is hard. I'll try to spare a few more brain cells when I'm back in the code.

Reply:
I changed it to reserved so that there's no contention — I also updated our error messaging and the test checking that error messaging to match.